cloudinit/distros/freebsd.py

```diff
@@ -87,6 +88,9 @@ def add_user(self, name, **kwargs):
             if key in pw_useradd_opts and val and isinstance(val, str):
```

Comment:
Nit: we could fuse the new branch into the old first one by:

```python
if key in pw_useradd_opts and val and isinstance(val, (str, int)):
   pw_useradd_cmd.extend([pw_useradd_opts[key], val])
```

There is no need to convert the `int` to `str` as its `__str__` will be used in the command.

Reply:
I was trying to remember why I wrote it like that, and I think it's because of https://github.com/canonical/cloud-init/blob/main/cloudinit/subp.py#L288